webdriverio/web/webdriver/test/configs/wdio.saucelabs.android.conf.js

```diff
@@ -0,0 +1,67 @@
+const {config} = require('./wdio.shared.conf');
+const build = `Android Chrome WebdriverIO-V6 build-${new Date().getTime()}`
+
+// =========================
+// Sauce Simulator specific config
+// =========================
+config.user = process.env.SAUCE_USERNAME;
+config.key = process.env.SAUCE_ACCESS_KEY;
+// If you run your tests on Sauce Labs you can specify the region you want to run your tests
+// in via the `region` property. Available short handles for regions are `us` (default) and `eu`.
+// These regions are used for the Sauce Labs VM cloud and the Sauce Labs Real Device Cloud.
+// If you don't provide the region, it defaults to `us`.
+config.region = process.env.REGION || 'us';
+
+// ============
+// Capabilities
+// ============
+config.capabilities = [
+    /**
+     * Android Chrome
+     */
+    {
+        name: "Android 8.1 Web Test",
+        build: build,
+        browserName: 'Chrome',
+        platformName: 'Android',
+        platformVersion: '8.1',
+        deviceName: 'Android GoogleAPI Emulator',
+    },
+    {
+        name: "Android 9.0 Web Test",
+        build: build,
+        browserName: 'Chrome',
+        platformName: 'Android',
+        platformVersion: '9.0',
+        deviceName: 'Android GoogleAPI Emulator',
+    },
+    {
+        name: "Android 10 Web Test",
+        build: build,
+        browserName: 'Chrome',
+        platformName: 'Android',
+        platformVersion: '10.0',
+        deviceName: 'Android GoogleAPI Emulator',
+    },
+    {
+        name: "Android 11 Web Test",
+        build: build,
+        browserName: 'Chrome',
+        platformName: 'Android',
+        platformVersion: '11.0',
+        deviceName: 'Android GoogleAPI Emulator',
+    }
+
+];
+
+const appiumVersion = process.env.npm_config_appium_version
```

Comment:
Can you put the constant at the top of the file, it's more of a preferred way instead of a mandatory one

Reply:
moved to shared conf